src/Script/RamCalculations.ts

```diff
@@ -101,21 +110,19 @@ function parseOnlyRamCalculate(otherScripts: Map<ScriptFilePath, Script>, code:
     if (nextModule === undefined) throw new Error("nextModule should not be undefined");
     if (nextModule.startsWith("https://") || nextModule.startsWith("http://")) continue;
 
-    // Using root as the path base right now. Difficult to implement
-    const filename = resolveScriptFilePath(nextModule, root, ns1 ? ".script" : ".js");
-    if (!filename) {
-      return { errorCode: RamCalculationErrorCode.ImportError, errorMessage: `Invalid import path: "${nextModule}"` };
-    }
-    const script = otherScripts.get(filename);
+    const script = otherScripts.get(nextModule);
     if (!script) {
-      return { errorCode: RamCalculationErrorCode.ImportError, errorMessage: `No such file on server: "${filename}"` };
+      return {
+        errorCode: RamCalculationErrorCode.ImportError,
+        errorMessage: `File: "${nextModule}" not found on server: ${server}`,
+      };
     }
 
     parseCode(script.code, nextModule);
   }
 
   // Finally, walk the reference map and generate a ram cost. The initial set of keys to scan
-  // are those that start with __SPECIAL_INITIAL_MODULE__.
+  // are those that start with the name of the main script.
```

Comment:
What happens when the initial script is `/foo.js` and there is `/foo.js.bar.js`?

Reply:
```js
import { test } from "./foo.js.bar"
/** @param {NS} ns */
export async function main(ns) {
  ns.tprint(test(ns))
}
//foo.js.bar.js
export function test(ns) {
  return ns.getServer("home")
}
```

tried with this and it seems to just work?
i dont fully understand that lower section of the code where initialModule is used so if there is a special case you see that needs testing other than that let me know

Reply:
Yeah, I was legitimately asking because I don't know either. I'm vaguely concerned about this case, but I may have to look into it more.